src/proto/h2/client.rs

```diff
@@ -243,7 +242,9 @@ where
         if polled.is_ready() {
             *this.is_terminated = true;
         }
-        polled.map_err(|e| debug!("connection error: {}", e))
+        polled.map_err(|_e| {
+            debug!("connection error: {}", _e);
```

Comment:
while we're here, we may want to change these events to record the error as a structured field, like
```suggestion
            debug!(error = %_e, "connection error");
```
this isn't a big deal, but it could be worth doing, and we're changing all the existing tracing macros anyway...